.github/actions/nm-build-vllm/action.yml

```diff
@@ -30,14 +30,27 @@ runs:
       SUCCESS=0
       python setup.py bdist_wheel --dist-dir=dist --py-limited-api=cp38 || SUCCESS=$?
       echo "whl_status=${SUCCESS}" >> "$GITHUB_OUTPUT"
-      ls -alh dist
       WHL_FILEPATH=$(find dist -type f -iname "*linux_x86_64.whl")
       echo "whl: ${WHL_FILEPATH}"
+      # strip wheel
```

Comment:
let's try an avoid this sort of thing. i think it'll be better to do it "natively" through the build system instead of doing surgery after the fact.

Reply:
yep agreed, removed the strip part.